grove/fermion_transforms/jwtransform.py

```diff
@@ -0,0 +1,65 @@
+##############################################################################
+# Copyright 2016-2017 Rigetti Computing
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+"""
+Implementation of the Jordan-Wigner Transform
+"""
+from pyquil.paulis import PauliTerm
+
+
+class JW(object):
+
+    def __init__(self):
+        """
+        Jordan-Wigner object the appropriate Pauli operators
+        """
+
+    def create(self, index, boson=False):
+        """
+        Fermion creation operator at orbital 'n'
+
+        :param Int n: creation index
+        """
+        return self._operator_generator(index, 1.0, boson=boson)
+
+    def kill(self, index, boson=False):
+        """
+        Fermion annihilation operator at orbital 'n'
+
+        :param Int n: annihilation index
+        """
+        return self._operator_generator(index, 1.0, boson=boson)
```

Comment:
one of kill and create should have a different `conj` right? At present they make the same call